MAUI/Themes/Keys.md

```diff
@@ -294,4 +294,1497 @@ This page lists the keys for each control and the element to which it is mapped
+            SfCalendarSpecialDayIconColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of special day icon.
+            <br/>
+            <br/>
+        </td>
+    </tr>  	
+    <tr>
+    <td>
+            SfCalendarNormalBackground          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of calendar normal background.
+            <br/>
+            <br/>
+        </td>
+    </tr>  	
+</table>
+
+## SfBackdrop
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfBackdropStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfBackdropPageTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfBackdrop without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfBackdropPageTheme">CommonTheme</x:String>
+                <Color x:Key="SfBackdropPageFrontLayerBackgroundColor">Red</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+        <tr>
+        <td>
+            SfBackdropPageFrontLayerBackgroundColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+           The background color of the foreground layer on the backdrop page
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	</tr>
+</table>
+
+## SfBarcodeGenerator
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfBarcodeGeneratorStyles
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfBarcodeGeneratorTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfBarcodeGenerator without merging common theme resource and control style resource dictionaries.
```

Comment:
By merging this key in application resources, you can customize the appearance of SfBarcodeGenerator without merging common theme resources and control style resource dictionaries.